tests/run_magetab_writer.py

```diff
@@ -0,0 +1,53 @@
+# for testing
+import utils.converter_utils
+from utils.common_utils import create_logger, file_exists
+from utils.converter_utils import get_sdrf_path, guess_submission_type
+from converter.converting import data_objects_from_magetab
+from converter import datamodel2magetab
+from sys import argv
+import os
+import pandas
+
+
+def column_name_to_magetab(header):
+    """Transform unique column header back to MAGE-TAB style.
+    Column headers are expected to be separated by "~~~". """
+    header_list = header.split("~~~")
+    return header_list[-1]
+
+
+process_name = "magetab_writer"
+idf_file = argv[1]
```

Comment:
I would always suggest to avoid direct parameters parsing in favour of argparse or equivalent. This might seem quick initially, but if for whatever reason your cli requires more parameters, it becomes quickly unmanageable without the use or a proper cli argument parser.

Reply:
Completely agree. This was just thought a "quick and dirty" test script to run the new functions. But with your suggestion about integrated testing in mind, I've added some proper parameter handling and new option to specify the output directory.